client/types/charges.d.ts

```diff
@@ -41,7 +41,8 @@ export interface PaymentMethodDetails {
 		| 'ideal'
 		| 'p24'
 		| 'sepa_debit'
-		| 'sofort';
+		| 'sofort'
+		| 'klarna';
```

Comment:
Will you please create an issue for a similar change for Affirm and Afterpay once we are clear with this one?

Reply:
I don't follow you. Affirm/Afterpay is defined as first in the list.

Reply:
My bad, I over scrolled. I think this list is sorted, though, can we keep that approach and place `klarna` after `ideal`, where it belongs?